Source/WarpX.H

```diff
@@ -423,6 +438,10 @@ public:
          amrex::IntVect y_nodal_flag, amrex::IntVect z_nodal_flag,
          const int lev);
 
+    // Function to add prtl and cell contributions to costs_heuristic
+    void ComputeCostsHeuristic (int lev);
+    amrex::Real sumMultiFabOverLevs (const amrex::Vector<std::unique_ptr<amrex::MultiFab> >& VectorOfMultifabs);
```

Comment:
please add a doxygen string describing what this abbreviated name does and what the args and side-effects of its call are.

Reply:
Documentation added.